### ISSUE TITLE
[Documentation]: Broken links on docs page

### ISSUE THREAD

**Issue by adactio** [storybookjs/storybook#28369]
### Describe the problem

On the landing page of the docs:

<https://storybook.js.org/docs>

There's a link to "Check out the installation guide". That links to:

<https://storybook.js.org/get-started/install>

That's broken. It should be:

<https://storybook.js.org/docs/get-started/install>

I tried to fix this by clicking on the "Edit on Github" link, but that points to:

<https://github.com/storybookjs/storybook/tree/next/docs.mdx>

…which is also broken.

**Comment by jonniebigodes** [comment]
Thanks for taking the time to reach out to us with your issue. We appreciate it 🙏 ! We're doing some maintenance documentation-wise, and there's still some QA left to be addressed to get it working correctly. Nonetheless, I'll reach out to the rest of the maintainers to check what's happening with the contribution link and hopefully get it to work. 


Hope you have a great day.

Stay safe

**Comment by kylegach** [comment]
Hi, @adactio —

The broken links should now be fixed. Thanks again for reporting the issue!

The broken "Edit on GitHub" link is a related issue and will be addressed soon. I wanted to fix the content links asap.

-----

If you're curious, we just launched a new Storybook site, which now uses index pages in the docs content where there previously were none. The links in our docs content need to work not only in the site, of course, but also when rendered as markdown on GitHub or in an editor, like VS Code on your machine. We failed to account for links on an index page requiring a bit of [finagling](https://github.com/storybookjs/web/commit/95465d8d497e5c846800eba842dfe7e1479347f5) to work in all of these contexts.

I know how important avoiding link rot is to you. While I regret the error made, it was fun to be able to respond to an issue you reported!
